russian/wca-guidelines.md

```diff
@@ -71,7 +73,7 @@
 - 3h++) [ПРИМЕР] Примеры совершенствования основной структуры: возможны новые движения, невозможны обычные движения, видно больше элементов или граней, видны цвета на задней стороне, движения выполняются автоматически, больше или другие собранные состояния.
 - 3h2++) [УТОЧНЕНИЕ] Головоломки "из цветного пластика", которые существенно отличаются от большинства головоломок "из цветного пластика" серийного производства, можно использовать только по решению Делегата WCA.
 - 3j+) [НАПОМИНАНИЕ] Раньше было разрешено использовать головоломки с элементами, имеющими гравировки или рельефы. Теперь это запрещено.
-- 3j++) [УТОЧНЕНИЕ] В Clock'е кнопки не должны быть отличимы от других кнопок на той же стороне.
+- 3j++) [УТОЧНЕНИЕ] В Clock'е кнопки не должны могут отличаться от других кнопок на той же стороне.
```

Comment:
I think there is a typo, did you mean this:

```suggestion
- 3j++) [УТОЧНЕНИЕ] В Clock'е кнопки могут отличаться от других кнопок на той же стороне.
```

Reply:
Thanks! I will fix it.

Reply:
Thanks! I will fix it.